#!/home/sergembela/anaconda3/envs/cdr_telecom/bin/python3.11
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
